### ISSUE TITLE
False positive indent after updating from 0.47.1

### ISSUE THREAD

**Issue by piotrp** [pinterest/ktlint#1830]
## Expected Behavior

Following code should pass indent validation:
```kotlin
    private fun test(): Boolean? =
        runCatching { true }
            .getOrNull()?.let { result ->
                !result
            }
```

It's also how IDEA indents code.

## Observed Behavior

0.48.2 and current master ([44ad030](https://github.com/pinterest/ktlint/commit/44ad0303546f07887058ed444a73cd9c1df1e712)) return:

```
[LintError(line=4, col=1, ruleId=standard:indent, detail=Unexpected indentation (12) (should be 8), canBeAutoCorrected=true),
    LintError(line=5, col=1, ruleId=standard:indent, detail=Unexpected indentation (8) (should be 4), canBeAutoCorrected=true)]
```

This snippet was accepted by 0.47.1.
